lib/_http_server.js

```diff
@@ -213,8 +213,25 @@ ServerResponse.prototype.writeHeader = function() {
 };
 
 
-function Server(requestListener) {
-  if (!(this instanceof Server)) return new Server(requestListener);
+var TLS_ARGS = ['pfx', 'key', 'passphrase', 'cert', 'ca', 'crl', 'ciphers',
```

Comment:
this seems brittle to me, perhaps we could consider connecting it with the https module such that when the options are modified there we don't run the risk of this falling out of sync?